cutting_board_empty_1:
  child_frame_id: cutting_board_empty_1
  fixed: false
  frame_id: world
  pose:
  - 0.5
  - -0.4
  - 0.48000000000000004
  - 0
  - 0
  - 0
  - 1
  type: cutting_board
  area: [cutting_board_ingredient_area]
cutting_board_empty_2:
  child_frame_id: cutting_board_empty_2
  fixed: false
  frame_id: world
  pose:
  - 0.5
  - -0.5800000000000001
  - 0.48000000000000004
  - 0
  - 0
  - 0
  - 1
  type: cutting_board
  area: [cutting_board_ingredient_area]
deactivate:
  child_frame_id: deactivate
  fixed: true
  frame_id: world
  pose:
  - 0
  - 4
  - 4
  - 0
  - 0
  - 0
  - 1
  type: deactivate
  area: [deactivate]
stove_1:
  child_frame_id: stove_1
  fixed: false
  frame_id: world
  pose:
  - 0.5
  - 2.125
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: stove
  area: [stove_cookware_area]
stove_2:
  child_frame_id: stove_2
  fixed: false
  frame_id: world
  pose:
  - 0.5
  - 1.875
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: stove
  area: [stove_cookware_area]
table1_bowl:
  child_frame_id: table1_bowl
  fixed: true
  frame_id: world
  pose:
  - 0.625
  - 0.875
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_dishware_area]
table1_cutting_board:
  child_frame_id: table1_cutting_board
  fixed: true
  frame_id: world
  pose:
  - 0.5
  - -0.49000000000000005
  - 0.47000000000000003
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_cutting_board]
table1_empty_1:
  child_frame_id: table1_empty_1
  fixed: false
  frame_id: world
  pose:
  - 0.625
  - 0.08499999999999996
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_dishware_area]
table1_empty_2:
  child_frame_id: table1_empty_2
  fixed: false
  frame_id: world
  pose:
  - 0.625
  - -0.16500000000000004
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [near_cutting_board_bowl_area, table1_dishware_area]
table1_empty_3:
  child_frame_id: table1_empty_3
  fixed: false
  frame_id: world
  pose:
  - 0.375
  - 0.08499999999999996
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_dishware_area]
table1_empty_4:
  child_frame_id: table1_empty_4
  fixed: false
  frame_id: world
  pose:
  - 0.375
  - -0.16500000000000004
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [near_cutting_board_bowl_area, table1_dishware_area]
table1_ing_1:
  child_frame_id: table1_ing_1
  fixed: true
  frame_id: world
  pose:
  - 0.6699999999999999
  - 0.66
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_ing_2:
  child_frame_id: table1_ing_2
  fixed: true
  frame_id: world
  pose:
  - 0.6699999999999999
  - 0.48
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_ing_3:
  child_frame_id: table1_ing_3
  fixed: true
  frame_id: world
  pose:
  - 0.6699999999999999
  - 0.30000000000000004
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_ing_4:
  child_frame_id: table1_ing_4
  fixed: true
  frame_id: world
  pose:
  - 0.5
  - 0.66
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_ing_5:
  child_frame_id: table1_ing_5
  fixed: true
  frame_id: world
  pose:
  - 0.5
  - 0.48
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_ing_6:
  child_frame_id: table1_ing_6
  fixed: true
  frame_id: world
  pose:
  - 0.5
  - 0.30000000000000004
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_ing_7:
  child_frame_id: table1_ing_7
  fixed: true
  frame_id: world
  pose:
  - 0.32999999999999996
  - 0.66
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_ing_8:
  child_frame_id: table1_ing_8
  fixed: true
  frame_id: world
  pose:
  - 0.32999999999999996
  - 0.48
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_ing_9:
  child_frame_id: table1_ing_9
  fixed: true
  frame_id: world
  pose:
  - 0.32999999999999996
  - 0.30000000000000004
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ingredient_area]
table1_knife:
  child_frame_id: table1_knife
  fixed: true
  frame_id: world
  pose:
  - 0.35
  - -0.8
  - 0.45
  - -0.70710678
  - 0
  - 0
  - 0.70710678
  type: table1
  area: [table1_tool_area]
table1_mayonnaise:
  child_frame_id: table1_mayonnaise
  fixed: true
  frame_id: world
  pose:
  - 0.65
  - -1.4000000000000001
  - 0.5
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ing_bottle_area]
table1_pan:
  child_frame_id: table1_pan
  fixed: true
  frame_id: world
  pose:
  - 0.5
  - 1.375
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_cookware_area]
table1_pepper:
  child_frame_id: table1_pepper
  fixed: true
  frame_id: world
  pose:
  - 0.65
  - -1.2000000000000002
  - 0.5
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ing_bottle_area]
table1_plate:
  child_frame_id: table1_plate
  fixed: true
  frame_id: world
  pose:
  - 0.375
  - 0.875
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_dishware_area]
table1_pot:
  child_frame_id: table1_pot
  fixed: true
  frame_id: world
  pose:
  - 0.5
  - 1.125
  - 0.4
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_cookware_area]
table1_salt:
  child_frame_id: table1_salt
  fixed: true
  frame_id: world
  pose:
  - 0.65
  - -1.0
  - 0.5
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ing_bottle_area]
table1_scooper:
  child_frame_id: table1_scooper
  fixed: true
  frame_id: world
  pose:
  - 0.35
  - -1.0
  - 0.45
  - -0.70710678
  - 0
  - 0
  - 0.70710678
  type: table1
  area: [table1_tool_area]
table1_spatula:
  child_frame_id: table1_spatula
  fixed: true
  frame_id: world
  pose:
  - 0.35
  - -1.4000000000000001
  - 0.45
  - -0.70710678
  - 0
  - 0
  - 0.70710678
  type: table1
  area: [table1_tool_area]
table1_spreader:
  child_frame_id: table1_spreader
  fixed: true
  frame_id: world
  pose:
  - 0.35
  - -1.2000000000000002
  - 0.45
  - -0.70710678
  - 0
  - 0
  - 0.70710678
  type: table1
  area: [table1_tool_area]
table1_sugar:
  child_frame_id: table1_sugar
  fixed: true
  frame_id: world
  pose:
  - 0.65
  - -0.8
  - 0.5
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ing_bottle_area]
table1_relish:
  child_frame_id: table1_relish
  fixed: true
  frame_id: world
  pose:
  - 0.65
  - -1.6000000000000001
  - 0.5
  - 0
  - 0
  - 0
  - 1
  type: table1
  area: [table1_ing_bottle_area]
